platform/admin/admin-configuration-configinstaller/src/main/java/org/codice/ddf/admin/configuration/ConfigurationInstaller.java

```diff
@@ -18,13 +18,19 @@
 
```

Comment:
❗️ Are we guaranteed that ConfigAdmin will not call this listener multiple times at the same time from different threads? If not then we should be careful about synchronization as we check and update internal attributes in the same method at different places.